apps/console/src/features/authentication-flow-builder/components/predefined-flows-side-panel/predefined-flows-side-panel.tsx

```diff
@@ -210,7 +210,8 @@ const PredefinedFlowsSidePanel: FunctionComponent<PredefinedFlowsSidePanelPropsI
 
         const sequence: AuthenticationSequenceInterface = {
             ...template.sequence,
-            script: AdaptiveScriptUtils.generateScript(template.sequence.steps.length + 1).join("\n")
+            script: template.sequence.script ? template.sequence.script : 
```

Comment:
We can use the `??` operator to simplify the code. Please check if this works.
```suggestion
            script: template.sequence.script ??AdaptiveScriptUtils.generateScript(template.sequence.steps.length + 1).join("\n")
```

Reply:
Resolved via https://github.com/wso2/identity-apps/pull/4531/commits/355ad27aa43d2d7d124bd23c02b5e9ff40ca200a